src/pspm_overwrite.m

```diff
@@ -0,0 +1,89 @@
+function ow_final = pspm_overwrite(varargin)
+
+% DESCRIPTION
+% pspm_overwrite generalises the overwriting operation
+% pspm_overwrite considers the following situations
+% - whether options.overwrite is defined
+% - whether PsPM is in develop mode
+% - whether the file exist
+% - whether GUI can be used
+% Only the following situation will stop PsPM to overwrite files
+% 1. if overwrite is defined as not to overwrite
+% 2. if overwrite is not defined, and users use the GUI to stop overwriting
+
```

Comment:
Wouldn't this blank line (without a `%` at the beginning) prevent the documentation to be correctly generated?
With and without a `%` it gives me a full or a truncated help section.
![image](https://user-images.githubusercontent.com/10925826/162928387-0bc33349-0243-4b7f-bd80-40dc8e652c02.png)
![image](https://user-images.githubusercontent.com/10925826/162928473-a978b148-5df7-4ee5-884d-e3cfeb2a89dc.png)

Reply:
Yes... I have deleted that blank line break.